release notes/v0.8.0.md

```diff
@@ -0,0 +1,43 @@
+
+
+## Experimental Module Merge into k6
+
+We have refactored code in preparation for the merge into `k6`. This will mean that all future releases of `xk6-browser` will also be part of the `k6` releases (as an experimental module). Merging it into `k6` will be super useful for you and us as it will increase the reach and visibility of the tool, which should mean more feedback and guidance from the community to allow us to tailor the product to your needs. It's worth noting that we're not merging the two code bases into a single one.
+
+As an experimental module, it still means that the APIs are likely to change, causing breaking changes in future releases -- for example, we're going to move more of our APIs to `async`, now that we have `async`/`await` keyword support, but this also means that your Playwright scripts will also start to work in `xk6-browser` with less and less tweaking of your test scripts. Another point to make here is that we're hoping for rough compatibility of Playwright APIs, so we will support some or all of Playwrights APIs, and also extend them too.
+
+This still means that browser based tests can only be performed within your own test environments, and it won't be available in the [cloud](https://k6.io/cloud/) (yet). Keep your eyes peeled though on future releases, and the blog to hear more on the public launch of browser testing in the cloud.
+
+With this release we are continuing to build and upload the artifacts (binaries) for you to download and work with. In the near future we intend to stop building the binaries and uploading them here, and instead you will be redirected to the k6 repo, which will have the browser test functionality in experimental form. We may also stop releasing the release notes here and move them to the k6 release notes.
+
+
+## Improvements
+
+- Convert promisified (`.then()`) examples to `async`/`await`. [#703](https://github.com/grafana/xk6-browser/pull/703)
+
+
+### Documentation
+
+- We're in the process of updating all our docs so that they reflect the merge and the `async`/`await` changes. These updates will be released after the merge and the latest version of `k6` has been released.
+
+
+## Internals
+
+- Upgraded `k6` dependency to `?`. ([#?](https://github.com/grafana/xk6-browser/pull/?))
+- Several refactors and bug fixes to improve maintainability and stability. ([#694](https://github.com/grafana/xk6-browser/pull/694), [#696](https://github.com/grafana/xk6-browser/pull/696), [#699](https://github.com/grafana/xk6-browser/pull/699), [#700](https://github.com/grafana/xk6-browser/pull/700), [#712](https://github.com/grafana/xk6-browser/pull/712), [#713](https://github.com/grafana/xk6-browser/pull/713), [#714](https://github.com/grafana/xk6-browser/pull/714), [#716](https://github.com/grafana/xk6-browser/pull/716), [#723](https://github.com/grafana/xk6-browser/pull/723))
```

Comment:
Add https://github.com/grafana/xk6-browser/issues/729 here if we merge it in.

Reply:
Done in [e1fb333](https://github.com/grafana/xk6-browser/pull/724/commits/e1fb33358c94cff095c484154364adfbbaa064d7).